main.js

```diff
@@ -0,0 +1,46 @@
+//
+// let diceOne = {
+//   val: () => Math.floor((Math.random() * 6) + 1),
+//   roll: () => {
+//     document.querySelector('.d1 div').className = `dice-${diceOne.val()}`
+//   }
+// }
+//
+// let diceTow = {
+//   val: () => Math.floor((Math.random() * 6) + 1),
+//   roll: () => {
+//     document.querySelector('.d2 div').className = `dice-${diceTow.val()}`
+//   }
+// }
+//
+// document.querySelector('.botton').addEventListener('click', () => {
+//   diceOne.roll()
+//   diceTow.roll()
+// })
+
+function getRandom(min, max) {
+   min = Math.ceil(min);
+   max = Math.floor(max);
+   return Math.floor(Math.random() * (max - min + 1)) + min;
+}let dieOne = {
```

Comment:
Please make your code more readable by creating new declarations and statements in new lines.